app-ng/src/entry-point.lisp

```diff
@@ -16,19 +83,37 @@
   (format t "~A [~A]~%" +QVM-VERSION+ +GIT-HASH+))
 
 (defun show-welcome ()
-  (format t "~&~
-*****************************************~%~
-* Welcome to the Rigetti QVM (Next Gen) *~%~
-*****************************************~%~
+  (format *error-output* "~&~
+******************************************
+* Welcome to the Rigetti QVM (Next Gen.) *~%~
+******************************************~%~
 Copyright (c) 2016-2019 Rigetti Computing.~2%")
-  (format t "(Configured with ~A MiB of workspace and ~D core~:P.)~2%"
+  (format *error-output* "(Configured with ~A MiB of workspace and ~D worker~:P.)~%"
           #+sbcl
           (floor (sb-ext:dynamic-space-size) (expt 1024 2))
           #-sbcl
           "many many"
```

Comment:
This message was inherited from the existing qvm-app.  I'm actually not sure if qvm(-app) supports non-SBCL. It depends on cl-quil, and I don't think that compiles elsewhere, but I could be wrong. For sure the build script that produces the qvm / qvm-ng binaries is SBCL only, but course it would be cool to support other impls, and in general we try to be portable where possible. I assume it wouldn't be too hard to track down similar API for at least the most common non-sbcl impls and Do The Right Thing here.